protocols/player.go

```diff
@@ -0,0 +1,7 @@
+package protocols
+
+type Player interface {
+	ID() string
+	Name() (*string, error)
+	LocalizedName(locale Locale) (*string, error)
+}
```

Comment:
**[QUESTION]:** `Player` interface does not need anymore methods `RefID()` and `IconPath()`? Also original `Names()` was changed to `Name()`. I just want to ensure this is changed by design.

Reply:
F2F